docker/back/api/nginx/vhost.conf

```
@@ -0,0 +1,25 @@
+server_tokens off;
+
+log_format docker '$remote_addr - [$time_local] "$request" $status $body_bytes_sent "$http_referer" "$http_user_agent" $request_time "$http_x_forwarded_for"';
+
+server {
+    root /var/www/html/public;
+
+    access_log /var/log/nginx/access.log docker;
+    error_log /var/log/nginx/error.log error;
+
+    location / {
+        try_files $uri /index.php$is_args$args;
+    }
+    location ~ ^/index.php(/|$) {
+        fastcgi_pass back_api_php:9000;
+        fastcgi_split_path_info ^(.+\.php)(/.*)$;
+        include fastcgi_params;
```

```suggestion
        include fastcgi_params;
        fastcgi_param HTTP_X_REQUEST_ID $request_uid;
```
Transfer l'identifiant de requête à PHp

---

done